pkg/ssh/client/ssh.go

```diff
@@ -32,13 +33,11 @@ func LoadKeyFromFile(filePath, clusterName, ngName string, ec2API ec2iface.EC2AP
 	}
 
 	key := string(fileContent)
-
-	fingerprint, err := pki.ComputeAWSKeyFingerprint(key)
+	fingerprint, err := fingerprint(filePath, fileContent)
```

Comment:
Interesting that this wasn't picked up by golangci-lint. I guess we have that check disabled.

You are not checking the `err` but using the `fingerprint` result immediately. So first, check the `err`. :)

Reply:
oh yeah. Fixed